cfg/VideoStream.cfg

```diff
@@ -23,14 +23,15 @@ gen.add("flip_horizontal", bool_t, LEVEL.NORMAL, "Flip image horizontally", Fals
 gen.add("flip_vertical", bool_t, LEVEL.NORMAL, "Flip image vertically", False)
 gen.add("width", int_t, LEVEL.RUNNING, "Target width", 0, 0, 10000)
 gen.add("height", int_t, LEVEL.RUNNING, "Target height", 0, 0, 10000)
-gen.add("brightness", double_t, LEVEL.RUNNING, "Target brightness", 0.0, 0.0, 1.0)
+gen.add("brightness", double_t, LEVEL.RUNNING, "Target brightness", 0.5, 0.0, 1.0)
```

Comment:
I'm making the default brightness of image capture at `0.5`, which is usually a good starting point (the previous value of 0.0 is usually dark)